rocketpy/simulation/flight.py

```diff
@@ -2113,88 +2069,94 @@ def e3(self):
+        time. Sometimes referred to as yaw acceleration."""
         return self.__evaluate_post_process[:, [0, 5]]
 
     @funcify_method("Time (s)", "α3 (rad/s²)", "spline", "zero")
     def alpha3(self):
-        """Rocket angular acceleration α3 as a Function of time."""
+        """Angular acceleration of the rocket in the z-axis as a Function of
+        time. Sometimes referred to as roll acceleration."""
         return self.__evaluate_post_process[:, [0, 6]]
 
     # Process third type of outputs - Temporary values
     @funcify_method("Time (s)", "R1 (N)", "spline", "zero")
     def R1(self):
-        """Aerodynamic force along the first axis that is perpendicular to the
-        rocket's axis of symmetry as a Function of time."""
+        """Aerodynamic force along the x-axis of the rocket as a Function of
+        time."""
         return self.__evaluate_post_process[:, [0, 7]]
 
     @funcify_method("Time (s)", "R2 (N)", "spline", "zero")
     def R2(self):
-        """Aerodynamic force along the second axis that is perpendicular to the
-        rocket's axis of symmetry as a Function of time."""
+        """Aerodynamic force along the y-axis of the rocket as a Function of
+        time."""
         return self.__evaluate_post_process[:, [0, 8]]
 
     @funcify_method("Time (s)", "R3 (N)", "spline", "zero")
     def R3(self):
-        """Aerodynamic force along the rocket's axis of symmetry as a
+        """Aerodynamic force along the z-axis (rocket's axis of symmetry) as a
         Function of time."""
         return self.__evaluate_post_process[:, [0, 9]]
 
-    @funcify_method("Time (s)", "M1 (Nm)", "spline", "zero")
+    @funcify_method("Time (s)", "M1 (Nm)", "linear", "zero")
     def M1(self):
-        """Aerodynamic bending moment in the same direction as the axis that is
-        perpendicular to the rocket's axis of symmetry as a Function of
-        time.
-        """
+        """Aerodynamic moment in the rocket x-axis as a Function of time.
+        Sometimes referred to as pitch moment."""
         return self.__evaluate_post_process[:, [0, 10]]
 
-    @funcify_method("Time (s)", "M2 (Nm)", "spline", "zero")
+    @funcify_method("Time (s)", "M2 (Nm)", "linear", "zero")
     def M2(self):
-        """Aerodynamic bending moment in the same direction as the axis that is
-        perpendicular to the rocket's axis of symmetry as a Function
-        of time."""
+        """Aerodynamic moment in the rocket y-axis as a Function of time.
+        Sometimes referred to as yaw moment."""
         return self.__evaluate_post_process[:, [0, 11]]
 
-    @funcify_method("Time (s)", "M3 (Nm)", "spline", "zero")
+    @funcify_method("Time (s)", "M3 (Nm)", "linear", "zero")
     def M3(self):
-        """Aerodynamic bending moment in the same direction as the rocket's
-        axis of symmetry as a Function of time."""
+        """Aerodynamic moment in the rocket z-axis as a Function of time.
```

Comment:
any reason for removing the bending term? Was the description wrong? just trying to understand

```suggestion
        """Aerodynamic bending moment in the rocket z-axis as a Function of time.
```

Reply:
Bending moment is usually used for solid mechanics analysis and might be misleading in this case since we are just calculating the moment generated by aerodynamic forces.

It might still be equal to the bending moment, but that could lead to the thought that something related to solid mechanics was involved in its calculation, which it wasn't